crates/apollo-router/Cargo.toml

```diff
@@ -69,6 +68,7 @@ httpmock = "0.6.2"
 insta = "1.8.0"
 maplit = "1.0.2"
 mockall = "0.10.2"
+parking_lot = "0.11.2"
```

Comment:
I suppose that one is for the counter in the integration test? Maybe we use the std's one and use unwrap to remove the burden of maintaining a dependency just for a test?

Reply:
good point,it'll be easier that way

Reply:
d16b35d